### PR TITLE
RU : Add some channels

### PR DESCRIPTION
**Pull request by BellezaEmporium**
vlcsnap-2021-10-04-13h44m44s768

vlcsnap-2021-10-04-13h47m26s706

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
vlcsnap-2021-10-04-13h53m51s689

If a Russian speaker could tell me the name of this channel.

**Review by dct-infra — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
@freearhey or @Dum4G when one of you will be available, please check the different names on the pictures. I sadly do not have any cyrillic alphabet knowledge.

File: channels/ru.m3u [hunk 1]
```diff
@@ -426,6 +432,8 @@ https://strm.yandex.ru/kal/muztv_supres/muztv_supres0.m3u8
 #EXTINF:-1 tvg-id="MuzTV.ru" tvg-country="RU" tvg-language="Russian" tvg-logo="https://i.imgur.com/n7bGFXj.png" group-title="Music",Муз-ТВ (480p)
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148
 http://ott-cdn.ucom.am/s28/index.m3u8
+#EXTINF:-1 tvg-id="Muz1TV.ru" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Music",Музыка Первого
```
Comment (freearhey) [inline channels/ru.m3u:435]:
```suggestion
#EXTINF:-1 tvg-id="MuzikaPervogo.ru" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Music",Музыка Первого
```

**Comment by freearhey** [comment]
1. Время (https://www.lyngsat.com/tvchannels/ru/Vremya.html)
2. Телекафе (https://www.lyngsat.com/tvchannels/ru/Telecafe.html)
3. Карусель (https://www.lyngsat.com/tvchannels/ru/Karusel.html)

**Review by freearhey — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
> 
> 
>     1. Время (https://www.lyngsat.com/tvchannels/ru/Vremya.html)
> 
>     2. Телекафе (https://www.lyngsat.com/tvchannels/ru/Telecafe.html)
> 
>     3. Карусель (https://www.lyngsat.com/tvchannels/ru/Karusel.html)

Thank you, channels added.

**Review by freearhey — APPROVED** [review]

**Review by Dum4G — APPROVED** [review]

**Pull request merged**